src/content/docs/dropins/product-details/pdp-initialization.mdx

```diff
@@ -0,0 +1,96 @@
+---
+title: PDP dropin initialization
+description: Learn about the configuration options for the PDP initializer.
+---
+
+import Diagram from '@components/Diagram.astro';
+import Callouts from '@components/Callouts.astro';
+import OptionsTable from '@components/OptionsTable.astro';
+import Aside from '@components/Aside.astro';
+import { Steps } from '@astrojs/starlight/components';
+
+Customizing the PDP dropin initializer can help you meet your project requirements and use cases.
+
+## Configuration options
+
+The PDP dropin initializer allows you to define the language definitions, default locale, and models that your project will use.
+
+<OptionsTable
+  options={[
+    ['Option', 'Type', 'Req?', 'Description'],
+    ['langDefinitions', 'object', 'No', 'Provides language definitions for internationalization (i18n).'],
+    ['defaultLocale', 'string', 'No', 'Specifies the default locale. If not provided, defaults to en-US.'],
+    ['models', 'object', 'No', 'Defines the data structure and initial data. Options include initialData, transform, and fallBackData.'],
+  ]}
+/>
+
+:::note
+Models are used to define the data structure and initial data for the PDP dropin. The `initialData` object contains product details, options, and attributes.
+:::
+
+## Example
+
+The following code shows the default implementation of the PDP dropin initializer in this Commerce boilerplate:
+
+```typescript
+// Define the models object with a ProductDetails property
+const models = {
+ProductDetails: {
+  initialData: { ...product },
+  },
+};
+
+// Initialize Dropins
+initializers.register(productApi.initialize, {
+  langDefinitions,
+  models,
+});
+```
+
+## Set default product options
+
+When a user navigates to the product detail page (PDP) on your site, you can set certain options as preselected defaults. This use case allows merchandisers to set default options through the Adobe Commerce Admin, which provides a more customized and streamlined shopping experience.
+
+* **Simple products**—Default option selection is supported out-of-the-box through the Admin (`isDefault`). No additional PDP dropin customization is required.
```

Comment:
I don't understand what you mean by this. A simple product is a simple product. It does not have any options, so there's no need for setting up any default options.

Reply:
My mistake @alandana 😅. I'm not sure what I was thinking there. I'll remove it.